package.json

```diff
@@ -50,6 +50,7 @@
     "posttest": "yarn lint",
     "prepack": "yarn run build",
     "test": "mocha --forbid-only \"test/**/*.test.ts\"",
+    "hooktest": "mocha --forbid-only \"test/typescript.test.ts\"",
```

Comment:
I don't love this. The `typescript.test.ts` has a test other than hooks too. And if you need to run one file, just run mocha directly. Let's remove this.